web/src/components/FeedbackButtons.tsx

```diff
@@ -19,23 +20,32 @@ type FeedbackButtonsProps = {
 
 const FeedbackButtons = ({ isPositive, onRatingChange }: FeedbackButtonsProps): ReactElement => {
   const { t } = useTranslation('feedback')
+
+  const handleChange = (event: React.MouseEvent<HTMLElement>, newValue: string | null) => {
+    if (newValue === t('useful')) {
+      onRatingChange(isPositive === true ? null : true)
+    } else if (newValue === t('notUseful')) {
+      onRatingChange(isPositive === false ? null : false)
+    }
+  }
+
+  const currentValue = () => {
+    if (isPositive === null) {
+      return null
+    }
+    if (isPositive) {
+      return t('useful')
+    }
+    return t('notUseful')
```

Comment:
🙃 
```suggestion
    return t(isPositive ? 'useful' : 'notUseful')
```

Reply:
I was trying to stay away from nested ternary.

Reply:
This is not a nested ternary, the null case is already handled before.